website/content/docs/upgrading/plugins.mdx

```diff
@@ -117,15 +115,28 @@ by setting the 'scope' parameter of the reload to 'global'.
         -version=v1.0.1 \
         database \
         my-db-plugin
+    Success! Registered plugin: my-db-plugin
     ```
 
-1. Update the database config with the new version. The database secrets
-   engine will immediately reload the plugin, using the new version. Any omitted
-   config parameters will not be updated.
+1. Set the new version as the cluster's pinned version.
 
     ```shell-session
-    $ vault write database/config/my-db \
-        plugin_version=v1.0.1
+    $ vault write sys/plugins/pins/database/my-db-plugin version=v1.0.1
+    ```
+
+1. Finally, trigger a global [plugin reload](/vault/docs/commands/plugin/reload)
+   to reload all instances of the plugin.
```

Comment:
BTW it looks like there's some sort of non-breaking space before "to reload all instances..." that may be causing the extra whitespace when rendered: https://vault-16n4u6u0z-hashicorp.vercel.app/vault/docs/upgrading/plugins#upgrading-database-plugins

Same thing on line 65 too: https://vault-16n4u6u0z-hashicorp.vercel.app/vault/docs/upgrading/plugins#upgrading-auth-and-secrets-plugins

Reply:
Good spot, I wonder if it's a bug to do with having a line break straight after a link. I've updated the line break location slightly, hopefully that helps.